backend/src/cli.py

```
@@ -0,0 +1,157 @@
+        image_dir=image_path,
+        output_dir=output_path,
+        model_name=model,
+        max_concurrent=concurrent,
+        max_images=max_images
+    ))
+    
+    if results:
+        completed = sum(1 for r in results if r["status"] == "completed")
+        failed = sum(1 for r in results if r["status"] == "failed")
+        
+        click.echo(f"\nSummary:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)